lib/active_fedora/containers/container.rb

```diff
@@ -0,0 +1,28 @@
+module ActiveFedora
+  class Container < ActiveFedora::Base
+
+    property :membership_resource, predicate: ::RDF::Vocab::LDP.membershipResource
```

Comment:
I don't understand how that would happen. We're always building it correctly, so why have a check that costs cycles and brings no benefit?